### PR TITLE
Add more methods for plotting borders

### LINKED ISSUE #385: 增加绘制国界省界汇总小节？
中文手册在多处提高不同的绘制国界和省界的方法和数据库：
- [入门教程：绘制国界/州界 (GSSHG)](https://docs.gmt-china.org/latest/tutorial/coastline/#id5)
- [DCW: 使用示例](https://docs.gmt-china.org/latest/dataset/dcw/#id6)
- [GADM: 全球行政区划数据库](https://docs.gmt-china.org/latest/dataset/gadm/#id4)
- [CN-border: 中国国界省界数据](https://docs.gmt-china.org/latest/dataset-CN/CN-border/)

[gmt-china.org/blog/2018-03-17-china-border-issues](https://github.com/gmt-china/gmt-china.org/edit/master/content/blog/2018-03-17-china-border-issues/index.md) 也提到此事:
> 目前已知的几套国界数据包括：
> - [GMT自带的GSHHG数据](https://docs.gmt-china.org/latest/dataset/gshhg/)：全球海岸线、国界、部分省/州界数据
> - [GMT自带的DCW数据](https://docs.gmt-china.org/latest/dataset/dcw/)：全球行政区划数据，包含洲界、国界、省界
> - [GADM数据](https://docs.gmt-china.org/latest/dataset/gadm/)
> - 社区提供的一套[相对准确的国界、省界数据](/example/ex003)

我觉得有必要将这些方式的链接汇总在一个文档种，方便用户查阅。暂时还没想好放在哪里。

### PR DESCRIPTION
**Pull request by core-man**
**Preview**: https://gmt-china.github.io/sitepreview/gmt-china/GMT_docs/borders/tutorial/coastline/#id5

Fixes #385.

### CODE REVIEW

File: source/tutorial/coastline.rst [hunk 1]
```diff
@@ -93,8 +93,9 @@ GMT中使用 :doc:`/module/coast` 模块绘制海岸线。
 绘制国界/州界
 -------------
 
-使用 **-N** 选项可以绘制国界/州界等行政边界。\ **-N1** 表示绘制国界线，
-**-N2** 表示绘制州界/省界线（目前只有美洲各国以及澳大利亚的国界的数据）。
+使用 **-N** 选项可以绘制国界、州界/省界等行政边界。\ **-N1** 表示绘制国界线，
+**-N2** 表示绘制州界/省界线。该选项绘制的行政边界来自 :doc:`/dataset/gshhg`\ ，
+目前只有美洲各国以及澳大利亚有州界/省界线。
```
Comment (core-man) [inline source/tutorial/coastline.rst:98]: 目前只有美洲各国以及澳大利亚有州界/省界线吧？

```
gmt coast -R-180/180/-89/89 -JM30c -Ba -Dh -A10000 -W1/0.5p -N1/thick,red -N2/thinner -pdf map
```

File: source/tutorial/coastline.rst [hunk 2]
```diff
@@ -30,8 +30,8 @@ GMT中使用 :doc:`/module/coast` 模块绘制海岸线。
 根据水体或陆地的不同，\ **coast**\ 模块将岸线分为从1到4的四个级别，
 依次指海岸线、湖岸线、湖中岛，以及湖中岛内的湖边界。
 
-因而，我们可以在使用 **-W** 选项时指定要绘制哪一个等级的岸线。下面的脚本中，
-我们使用 **-W1/0.5p,black** 表示用 0.5p 宽的黑色线条绘制1级海岸线。
+因而，我们可以在使用 **-W** 选项时指定要绘制哪一个等级的岸线。
+下面的脚本中，我们使用 **-W1/0.5p,black** 表示用 0.5p 宽的黑色线条绘制1级海岸线。
```
Comment (seisman) [inline source/tutorial/coastline.rst:34]:
```suggestion
下面的脚本中，我们使用 **-W1/0.5p,black** 表示用 0.5p 宽的黑色线条绘制 1 级海岸线。
```

File: source/tutorial/coastline.rst [hunk 3]
```diff
@@ -68,14 +68,14 @@ GMT中使用 :doc:`/module/coast` 模块绘制海岸线。
         gmt coast -R-130/-50/20/60 -JM15c -Baf -W1/0.5p,black -W2/1p,lightred -A5000
     gmt end show
 
-这里我们使用 **-A5000** 表示只绘制面积大于5000平方千米的湖泊或岛屿。这样子
-得到的图看上去更加干净简洁了。
+这里我们使用 **-A5000** 表示只绘制面积大于 5000 平方千米的湖泊或岛屿。
+这样子得到的图看上去更加干净简洁了。
```
Comment (seisman) [inline source/tutorial/coastline.rst:72]:
```suggestion
这样子得到的图看上去更加干净简洁。
```

File: source/tutorial/coastline.rst (hunk 1 above)
Comment (seisman) [reply source/tutorial/coastline.rst:98]: 是

File: source/tutorial/coastline.rst [hunk 4]
```diff
@@ -104,8 +105,14 @@ GMT中使用 :doc:`/module/coast` 模块绘制海岸线。
         gmt coast -R-130/-50/20/60 -JM15c -Baf -A5000 -Gred -Slightblue -Clightred -N1/1p -N2/0.25p
     gmt end show
 
-除了 **-N** 选项之外，\ **-E** 选项也可以用于绘制州界、国界和省界，在
-:doc:`/dataset/dcw/index` 一节会详细介绍。本入门教程中暂时略过。
+.. note::
```
Comment (seisman) [inline source/tutorial/coastline.rst:108]: 这里加 note，下面几行需要缩进。

Comment (core-man) [reply source/tutorial/coastline.rst:108]: done in https://github.com/gmt-china/GMT_docs/pull/626/commits/af9baa8f8da5b0b733c6483086d2b190d60c3afa.

File: source/tutorial/coastline.rst [hunk 5]
```diff
@@ -104,8 +105,14 @@ GMT中使用 :doc:`/module/coast` 模块绘制海岸线。
...
+
+    除了 **-N** 选项之外，还有其他方法使用 GMT 绘制洲界、国界、州界/省界等去行政边界：
```
Comment (seisman) [inline source/tutorial/coastline.rst:110]:
```suggestion
    除了 **-N** 选项之外，还有其他方法使用 GMT 绘制洲界、国界、州界/省界等行政区划边界：
```

**Review by seisman — APPROVED** [review]

File: source/tutorial/coastline.rst (hunk 1 above)
Comment (core-man) [reply source/tutorial/coastline.rst:98]: done in https://github.com/gmt-china/GMT_docs/pull/626/commits/55d529fdef446f9e719c815d19d7ea468605e69c

**Pull request merged**